src/OrchardCore.Modules/OrchardCore.Users/Module.txt

```diff
@@ -5,4 +5,5 @@ Website: http://orchardproject.net
 Version: 2.0.x
 OrchardVersion: 2.0.x
 Description: The users module enables authentication UI and user management.
+Dependencies: OrchardCore.Authentication
```

Comment:
@sebastienros should the `OrchardCore.Users` package also have a (NuGet) dependency on `OrchardCore.Authentication`?

Reply:
It's not a physical dependency so no. Even though this feature won't work without the other feature enabled. This just ensure that if one day we have another feature in the manifest that doesn't require it, then it won't download it for nothing.

Reply:
Well, if we don't transitively reference the authentication module, developers will have to do it themselves before being able to use the users module. If that's not a problem, that's fine.